scout/server/blueprints/variant/templates/variant/buttons.html

```diff
@@ -1,3 +1,7 @@
+{% macro splice_junctions_button(institute_id, case_name, variant_id) %}
+  <a class="btn btn-secondary text-white mt-1" href="{{url_for('alignviewers.sashimi_igv', institute_id=institute_id, case_name=case_name, variant_id=variant_id)}}" target="_blank"
+    data-toggle="tooltip" data-placement="top" title="Gene splicing is only available in build GRCh38">Gene splicing</a>
```

Comment:
Gene splicing makes my thoughts go to early biotech efforts. 😄  What do you think about "RNA splicing" or "Splice junctions"?

Reply:
Now that you mention it, yes! "RNA splicing" sounds good!